bentoml/_internal/configuration/containers.py

```diff
@@ -257,24 +233,31 @@ def tracer(
 
 
 @dataclass
-class BentoServerContainerClass:
+class DeploymentContainerClass:
 
     bentoml_container = BentoMLContainer
-    config = bentoml_container.config.bento_server
+    config = bentoml_container.config
```

Comment:
note: the tracing config should be and has been in the outer level of the API server. Runners share the same config.

Reply:
TODO: move metric config out of API server config as well?